manager-rs/dauth-service/src/local/manager.rs

```diff
@@ -61,57 +61,50 @@ pub async fn confirm_auth_vector_used(
             Err(DauthError::NotFoundError("Key not available".to_string()))
         }
     }
-
 }
 
 /// Returns whether the auth vector belongs to this core
 fn auth_vector_is_local(context: Arc<DauthContext>, av_request: &AuthVectorReq) -> bool {
-    (av_request.user_id <= context.local_context.local_user_id_max) &&
-    (av_request.user_id >= context.local_context.local_user_id_min)
+    (av_request.user_id <= context.local_context.local_user_id_max)
+        && (av_request.user_id >= context.local_context.local_user_id_min)
```

Comment:
This format with the leading && is kind of strange... is this the output from rustfmt?

Reply:
Yes, and I was confused because I noticed some other small changes like this too. I figured rustfmt would be basically consistent.

Reply:
Out of curiosity what version of rust and rustfmt are you using?

Reply:
rustc 1.56.1, cargo 1.56.0, rustfmt 1.4.37-stable